src/Raven.Studio/typescript/common/generalUtils.ts

```diff
@@ -468,7 +468,9 @@ class genUtils {
         if (!string) {
             return string;
         }
-        return $("<div/>").html(string).text();
+        const e = document.createElement("textarea");
```

Comment:
It would be nice to have a test showing that this case now works. Please also check the usage in studio to see if it breaks anything. Since this fixes something, we would like these changes from v5.4 and up